admin/src/components/Tables/OpenCreationsTable.vue

```diff
@@ -1,6 +1,21 @@
 <template>
   <div class="open-creations-table">
-    <b-table-lite :items="items" :fields="fields" caption-top striped hover stacked="md">
+    <b-table-lite
+      :items="items"
+      :fields="fields"
+      caption-top
+      striped
+      hover
+      stacked="md"
+      :tbody-tr-class="rowClass"
+    >
+      <template #cell(state)="row">
+        <b-icon v-if="row.item.state === 'IN_PROGRESS'" icon="question-square"></b-icon>
+        <b-icon v-if="row.item.state === 'PENDING'" icon="bell-fill"></b-icon>
+        <b-icon v-if="row.item.state === 'CONFIRMED'" icon="check"></b-icon>
+        <b-icon v-if="row.item.state === 'DELETED'" icon="trash"></b-icon>
+        <b-icon v-if="row.item.state === 'DENIED'" icon="x-circle"></b-icon>
```

Comment:
please create a map object and a method:
````const iconMap = {
  'IN_PROGRESS': 'question-square',
  'PENDING': 'bell-fill',
  ...
}
```
and
```
  getStatusIcon(status) {
    return iconMap[status] ? iconMap[status] : 'default-icon'
  }

Reply:
done